src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,42 +1,92 @@
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import moment from 'moment';
+
 import { focusElement } from '../../../common/utils/helpers';
 import OMBInfo from '../../../common/components/OMBInfo';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
-
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
 import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { submitIntentToFile } from '../../../common/schemaform/actions';
+
+import formConfig from '../config/form';
 
 class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      verify: false
+    };
+  }
+
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  handleLoadPrefill = () => {
+    // TODO: determine payload
+    submitIntentToFile('name/ssn/other', formConfig.intentToFileUrl, formConfig.trackingPrefix);
+    // TODO: store confirmation number in formData
```

Comment:
According to the swagger docs, the return value from the `POST` looks like this:
```json
{
  "messages": [
    {
      "key": "string",
      "text": "string",
      "severity": "FATAL"
    }
  ],
  "intentToFile": {
    "id": "string",
    "creationDate": "2018-04-04T03:18:36.507Z",
    "expirationDate": "2018-04-04T03:18:36.507Z",
    "participantId": 0,
    "source": "string",
    "status": "active",
    "type": "compensation"
  }
}
```
Not sure how much of that we want to store to show the user, but I don't think any of it has to be stored in the `formData`.